File: examples/Delphi/EXSRNMU2.pas
{form used for Serial Number/ReleaseCode input}
unit Exsrnmu2;

interface

uses
  SysUtils, WinTypes, WinProcs, Messages, Classes, Graphics, Controls,
  Forms, Dialogs, StdCtrls, Buttons;

type
  TSNEntryDlg = class(TForm)
    Label1: TLabel;
    Label2: TLabel;
    SNText: TEdit;
    CodeText: TEdit;
    OKBtn: TBitBtn;
    CancelBtn: TBitBtn;
    Label3: TLabel;
    procedure CancelBtnClick(Sender: TObject);
  private
    { Private declarations }
  public
    { Public declarations }
  end;

var
  SNEntryDlg: TSNEntryDlg;

implementation

{$R *.DFM}

procedure TSNEntryDlg.CancelBtnClick(Sender: TObject);
begin
  Close;
end;

end.

File: examples/CBuilder/EXSRNMU2.cpp
//---------------------------------------------------------------------------
#include <vcl\vcl.h>
#pragma hdrstop

#include "ExSrnmu2.h"
//---------------------------------------------------------------------------
#pragma resource "*.dfm"
TSNEntryDlg *SNEntryDlg;
//---------------------------------------------------------------------------
__fastcall TSNEntryDlg::TSNEntryDlg(TComponent* Owner)
	: TForm(Owner)
{
}
//---------------------------------------------------------------------------
void __fastcall TSNEntryDlg::CancelBtnClick(TObject *Sender)
{
	Close();	
}
//---------------------------------------------------------------------------